www/src/pegen.js

```diff
@@ -15,6 +15,11 @@
 var _b_ = __BRYTHON__.builtins
 
 const Load = new $B.ast.Load()
+const NULL = undefined;
+const ENDMARKER = 0,
```

Comment:
This was necessary to get the nodejs profiler running, but didn't seem necessary in the browser. I didn't have the time to dig into exactly why that discrepancy exists.